frontend/packages/component-library/src/cms/tabGroup/stories/TabGroup.story.tsx

```diff
@@ -0,0 +1,242 @@
+import {Meta, StoryObj} from '@storybook/react';
+import {within, expect} from '@storybook/test';
+
+import {LinkButtonProps} from '@/button';
+import {ImageProps} from '@/cms/image';
+
+import TabGroup, {TabGroupTabModel} from './../TabGroup';
+
+export default {
+  title: 'CMS/TabGroup',
+  component: TabGroup,
+  parameters: {
+    componentSubtitle: 'Renders a group of tabs with optional content',
+  },
+} as Meta<typeof TabGroup>;
+
+type Story = StoryObj<typeof TabGroup>;
+
+const defaultButton: LinkButtonProps = {
+  children: 'Click Me',
+  href: 'https://code.org',
+  text: 'Click me',
+};
+
+const defaultImage: ImageProps = {
+  src: 'https://code.org/images/teach-page-top.png',
```

Comment:
It might be worth hosting these images locally similar to how the Image component. You could even use the one that already exists. https://github.com/code-dot-org/code-dot-org/blob/5c6f6fef202b7b2075af2cb73e40c96583b7d504/frontend/packages/component-library/src/cms/image/stories/Image.story.tsx#L1

We will eventually deprecate these images once the site is moved over to Contentful.